src/lxc/tools/lxc_execute.c

```diff
@@ -160,7 +160,9 @@ int main(int argc, char *argv[])
 	bret = c->start(c, 1, my_args.argv);
 	ret = c->error_num;
 	lxc_container_put(c);
-	if (!bret)
+	if (!bret) {
```

Comment:
if `my_args.quiet` is true, the `ERROR` will output nothing to `stderr`.  so there is no need to add judgment